### ISSUE TITLE
Floated panels can be docked to the main window with UI edit mode disabled

### ISSUE THREAD

**Issue by runer112** [CE-Programming/CEmu#269]
If you drag a panel that is in a floating window on top of any other CEmu window while UI edit mode is disabled, the panel will become docked to the right side of the main window. This should not occur, as UI edit mode is disabled.

CEmu 1.1dev, 6cbb192, Windows 10.

**Comment by runer112** [comment]
```
10:04:48 <•MateoC> All of the docking issues are directly related to the "fix" of auto expanding docks inside the mainwindow
10:05:29 <Runer112> in other words... that fix is bugged and is breaking other things
10:05:36 <Runer112> these other things are not "impossible"
10:05:52 <•MateoC> Well, your thing is impossible
10:06:05 <Runer112> seems unlikely
10:06:08 <•MateoC> I spent like a week trying to fix it
10:06:11 <•MateoC> Never coud
10:06:14 <•MateoC> *could
10:06:27 <Runer112> that sounds far from proof that it's impossible
10:06:32 <•MateoC> :P
```

This is not impossible, it's just hard/tricky with the current docking implementation. I'm reopening the issue. Hopefully, in the future, someone will revisit it with fresh ideas and perhaps a different docking implementation and fix this.

**Comment by mateoconlechuga** [comment]
Fixed with b81d6b0
